docs/partitioning/README.md

```diff
@@ -0,0 +1,210 @@
+loop0     7:0    0  93.8G  0 loop /run/ephemeral
+loop1     7:1    0 897.3M  1 loop /sysroot
+sr0      11:0    1   999M  0 rom  /run/media/iso
+nvme0n1 259:1    0   1.5T  0 disk 
+```
+
+> :exclamation: It is recommended to use NVME disk in your servers.
+
+A second prerequisite is being able to pull the quay.io/openshift-kni/telco-ran-tools:latest that will be used to run the factory-cli tool. The image is publicly available in quay.io but if you are in a disconnected environment or have a corporate private registry, you will need to copy the image there so it can be downloaded to the server.
+
+```
+[root@snonode /]# podman pull quay.io/openshift-kni/telco-ran-tools:latest
+Trying to pull quay.io/openshift-kni/telco-ran-tools:latest...
+Getting image source signatures
+Copying blob 97da74cc6d8f done  
+Copying blob 8700ab5d88a1 done  
+Copying blob d8190195889e done  
+Copying blob b250919afbec done  
+Copying blob 5f6dcacfa818 done  
+Copying blob 6b1b76a0904e done  
+Copying blob cf95c38e62d8 done  
+Copying blob 86cf6373f6c4 done  
+Copying config 4b99f87ac7 done  
+Writing manifest to image destination
+Storing signatures
+4b99f87ac7872f8b6cdfea50ec0aed44d5145a33b171e22c33b866ce15759c72
+```
+
+```
+[root@snonode /]# podman run quay.io/openshift-kni/telco-ran-tools:latest -- factory-precaching-cli -v
+factory-precaching-cli version 20221018.120852+main.feecf17
+```
+
+Finally, we need to be sure that the disk is big enough to install OpenShift and also precache all the container images required: OCP release and day2 operators. Based on our experience:
+
+* If you are going to precache only OCP release artifacts a 100GB partition is enough
+* If you want to precache both OCP release artifacts and the DU profile day2 operators, around 250GB of disk space is required.
+* Lastly, take into account that OCP installation requires a minimum 120GB to be installed. So, you need to add this size to precached partition.
+
+> :warning: The factory-cli tool allows you to precache any other container image you will need for your environment. Then, make sure that the size of those extra images is taken into account in the sizing exercise.
+
+
+## Partitioning and formatting
+
+Let's start with the partitioning, for that we will use the partition argument from the factory-cli. See here all the options we have:
+
+```
+[root@snonode /]# podman run quay.io/openshift-kni/telco-ran-tools:latest -- factory-precaching-cli partition --help
+Partitions and formats a disk
+
+Usage:
+  factory-precaching-cli partition [flags]
+
+Flags:
+  -d, --device string   Device to be partitioned
+  -h, --help            help for partition
+  -s, --size int        Partition size in GB (default 100)
+  -v, --version         version for partition
+```
+
+### Verify the disk is cleared
+
+As mentioned in the previous section, we need our disk to be empty. Please make sure it is before starting the process. This is an example of a nvme0n1 device with no partitions:
```

Comment:
I believe the tool will fail if the disk isn't empty, as it hardcodes the partition number to 1. We could add a test to check it before attempting to write the partition, to provide a better error message, maybe